test/controllers/elastic_search_7_test.rb

```diff
@@ -300,5 +300,29 @@ def setup
     end
   end
 
+  test "should ignore index task responses that exceeds nested objects limit" do
+    team = create_team
+    tt = create_team_task team_id: team.id, type: 'single_choice', options: ['ans_a', 'ans_b', 'ans_c']
+    tt2 = create_team_task team_id: team.id, type: 'single_choice', options: ['ans_aa', 'ans_bb', 'ans_cc']
+    tt3 = create_team_task team_id: team.id, type: 'single_choice', options: ['ans_aaa', 'ans_bbb', 'ans_ccc']
+    pm = create_project_media team: team
+    stub_configs({ 'nested_objects_limit' => 2 }) do
+      # answer single choice
+      puts "Start to answer"
```

Comment:
Please delete this debugging line if you don't need it.

Reply:
OK will do